### PR TITLE
Settings Page

### PR DESCRIPTION
**Pull request by bhavitaap**
https://linear.app/bread-and-roses/issue/BRE-42/settings-page

created settings cards and hardcoded dummy info

[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- added settings cards for each of the subsections in the settings page
- hardcoded dummy information (based on the figma)

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- app/settings/page.tsx
- components/SettingsCard <- 5 new files created in this folder for each of the subsections
- expected behavior: when navigating to the Settings page, it should look like the Figma 
- feedback: is there possibly a way to collapse the files into one and use props instead of hardcoding the info

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- update the info from the database
- for bullet point info make it loop through relevant 
SettingsPageCards
info instead

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jxmoose

### CODE REVIEW

File: .env.local.save
Comment (jxmoose) [inline .env.local.save:1]: this shouldn't be added to git but also not sure if this file should exist in first place lol

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx:1]: can you change the px to rem! overall should almost always use rem over px. https://nekocalc.com/px-to-rem-converter u can use this website but for smaller ones just divide by 16 and it should be accurate.

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <styles.Edit src={Edit} alt="Edit" />
+      </styles.AvailabilityHeader>
+      <styles.Content>
+        <div>
+          <styles.SubHeader>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Type of Act
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx]: can you change the px to rem! overall should almost always use rem over px. https://nekocalc.com/px-to-rem-converter u can use this website but for smaller ones just divide by 16 and it should be accurate.

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                    Music
+                  </styles.TruncatedText>
+                </li>
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Genre
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx]: ^

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        <styles.Edit src={Edit} alt="Edit" />
+      </styles.AvailabilityHeader>
+      <styles.Content>
+        <div>
+          <styles.SubHeader>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Facility Type
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: ^

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                    Senior Homes
+                  </styles.TruncatedText>
+                </li>
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Location Preferences
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: ^

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+                    Oakland, CA
+                  </styles.TruncatedText>
+                </li>
+              </ul>
+            </styles.SettingDetail>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Preferred Audience
+              </P>
+              <ul style={{ paddingLeft: '20px', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: ^

File: components/SettingsCard/styles.ts [hunk 6]
```diff
@@ -44,21 +44,24 @@ export const Content = styled.main`
 export const SubHeader = styled.main`
   display: flex;
   flex-direction: column;
-  gap: 4px;
+  gap: 24px;
```
Comment (jxmoose) [inline components/SettingsCard/styles.ts]: ^

File: components/SettingsCard/styles.ts [hunk 7]
```diff
@@ -44,21 +44,24 @@ export const Content = styled.main`
...
   justify-content: start;
   margin-bottom: 0.25rem;
 `;
 
+export const SettingDetail = styled.main`
+  gap: 4px;
```
Comment (jxmoose) [inline components/SettingsCard/styles.ts]: ^

**Review by jxmoose — CHANGES_REQUESTED** [review]
overall looks great! just some small changes, if you look at the figma there some of the text should have bullet points if you could add those. also just general things never use px just use rem instead! also delete the .env.local.save file

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+        <styles.Edit src={Edit} alt="Edit" />
+      </styles.AvailabilityHeader>
+      <styles.Content>
+        <div>
+          <styles.SubHeader>
+            <styles.SettingDetail>
+              <P $fontWeight="500" $color={COLORS.gray12} $align="left">
+                Type of Act
+              </P>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx]: can you change these to not use in line styling and add a style component to styles.ts instead
